src/utilities/loadSampleData.ts

```diff
@@ -0,0 +1,98 @@
+import "dotenv/config";
+import yargs from "yargs";
+import { User, Organization, Event, Post } from "../models";
+import { connect } from "../db";
+import fs from "fs";
+import path from "path";
+
+// Required arguments that need to be parsed
+yargs.option("i", {
+  alias: "items",
+  describe: "Comma-separated list of collections to load sample data into",
+  type: "string",
+});
+
+yargs.option("f", {
+  alias: "format",
+  describe: "Whether to format the database before insertion",
```

Comment:
`Formats all the collections present in the database before the insertion of objects. [WARNING] Use carefully.`

This is a better description is my opinion.

Reply:
Yes. Would change it.